### PR TITLE
Browser+LibGUI: Add more tab options

### PR DESCRIPTION
**Pull request by SeekingBlues**
Add more tab options to the browser: "Duplicate Tab" and "Close Other Tabs".

### CODE REVIEW

File: Userland/Libraries/LibGUI/TabWidget.cpp [hunk 1]
```diff
@@ -64,6 +64,22 @@ void TabWidget::remove_widget(Widget& widget)
         on_tab_count_change(m_tabs.size());
 }
 
+void TabWidget::remove_all_tabs_other_than(Widget& widget)
```
Comment (awesomekling) [inline Userland/Libraries/LibGUI/TabWidget.cpp:67]: Let's call this `remove_all_tabs_except()`
Also please put it in a separate commit :)

**Review by awesomekling — CHANGES_REQUESTED** [review]

File: Userland/Libraries/LibGUI/TabWidget.cpp (hunk 1 above)
Comment (SeekingBlues) [reply Userland/Libraries/LibGUI/TabWidget.cpp]: Fixed and pushed a separate commit :^)

**Comment by awesomekling** [comment]
Please squash fixups into the preceding commit (see the [contributing guidelines](https://github.com/SerenityOS/serenity/blob/5d3b452897fd30b3b142b9aa4eaee5d0e01c367c/CONTRIBUTING.md)) :)

**Comment by SeekingBlues** [comment]
Of course. Done :)

File: Userland/Libraries/LibGUI/TabWidget.cpp [hunk 2]
```diff
@@ -64,6 +64,22 @@ void TabWidget::remove_widget(Widget& widget)
         on_tab_count_change(m_tabs.size());
 }
 
+void TabWidget::remove_all_tabs_except(Widget& widget)
```
Comment (awesomekling) [inline Userland/Libraries/LibGUI/TabWidget.cpp:67]: Could you please put the LibGUI change into a separate commit? :)

**Comment by SeekingBlues** [comment]
Sorry that I was a little bit confused. This should be all fixed now. :)
